drivers/storage/portworx/deployment.go

```diff
@@ -1464,6 +1460,20 @@ func (t *template) getK3sVolumeInfoList() []volumeInfo {
 	}
 }
 
+func (t *template) getPKSVolumeInfoList() []volumeInfo {
+	if !t.isPKS {
+		return []volumeInfo{}
+	}
+
+	return []volumeInfo{
+		{
+			name:      "containerd-pks",
+			hostPath:  "/var/vcap/sys/run/containerd/containerd.sock",
```

Comment:
If the containerd directory is under `/var/vcap/sys/run/containerd`, then why do we need to mount `/run/containerd` in PKS? Rather is `/run/containerd` even present on the PKS cluster?

Reply:
We'll actually need both --
* the `containerd` APIs are a little bit "weird" in a way that just the access to the socket-file is not sufficient, and we also need the access to the `/run/containerd` directory
   - (i.e. unlike DBus or Docker, where the only thing required for APIs to work is the socket-endpoint)
* as for the PKS specifically, those guys "broken up" their install, so they keep majority of the data in `/run/containerd`, but the socket is at `/var/vcap/sys/run/containerd/containerd.sock`